### DISCUSSION TITLE
Cron fails build even if cron is disabled for a project

### DISCUSSION THREAD

**Discussion by nathancahill** [vercel/community#1704] [Help] [answered]
### Summary

We're deploying the dev version of our app on the Hobby plan. We're using (and loving) the new cron feature on the Pro plan. On the Hobby plan, cron is disabled for the project, so all cron config in vercel.json should be ignored. However, we have a `* * * *` schedule, so we get this error:

Hobby accounts are limited to daily cron jobs. This cron expression (* * * * *) would run more than once per day. Upgrade to pro to unlock all Cron Jobs features on Vercel.

Cron is disabled completely for the project so I think the config should be ignored.

**Comment by amyegan** [comment, marked as answer]
Hi Nathan. [Disabled cron jobs still count](https://vercel.com/docs/cron-jobs#:~:text=Disabled%20cron%20jobs%20will%20still%20be%20listed%20and%20will%20count) toward your limits, so this is expected behavior. The limit checks are in place, even if Cron Jobs are disabled, to prevent a Hobby account from accidentally getting into an invalid state with the potential for more confusing errors.

Cron Jobs are still in [Beta](https://vercel.com/docs/concepts/release-phases#beta), so it's possible that this could change in the future as more safeguards are built. For now, Cron Jobs on Hobby plans must stay within the [limits](https://vercel.com/docs/cron-jobs#limits)
